config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'posts#index'
+
+  resources :users, only: [:index, :show] do
+  resources :posts, only: [:index, :show]
+
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to indent this code to enhance readability:

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
This is indented at my end as seen below:

![indented](https://user-images.githubusercontent.com/32777408/229614371-d14f09b7-aea3-4374-b028-ee7b80e3af0e.png)


Nevertheless. Thanks for the feedback